website/docs/cloud-docs/api-docs/policy-sets.mdx

```diff
@@ -568,7 +589,8 @@ Properties without a default value are required.
 | `data.attributes.vcs-repo.ingress-submodules` | boolean        | (previous value) | Determines whether repository submodules will be instantiated during the clone operation.                                                                                                                                                                                                                    |
 | `data.attributes.policies-path`               | boolean        | (previous value) | The subdirectory of the attached VCS repository that contains the policies for this policy set. Files and directories outside of this sub-path will be ignored, and changes to those unrelated files won't cause the policy set to be updated. This option may only be specified when a VCS repo is present. |
 | `data.relationships.projects`                 | array\[object] | (previous value) | An array of references to projects that the policy set should be assigned to. Sending an empty array clears all project assignments. You can only specify this attribute when `data.attributes.global` is `false`.                                                                                           |                                                                                                                                          |
-| `data.relationships.workspaces`               | array\[object] | (previous value) | An array of references to workspaces that the policy set should be assigned to. Sending an empty array clears all workspace assignments. You can only specify this attribute when `data.attributes.global` is `false`.                                                                                       |                                                                                                                                          |
+| `data.relationships.workspaces`               | array\[object] | (previous value) | An array of references to workspaces that the policy set should be assigned to. Sending an empty array clears all workspace assignments. You can only specify this attribute when `data.attributes.global` is `false`.                                                                                       | 
+| `data.relationships.exclusions`               | array\[object] | (previous value) | An array of references to workspaces that the policy set should not be assigned to. Sending an empty array clears all exclusions assignments. You can only specify this attribute when `data.attributes.global` is `false`.                                                                                      |                                                                                                                                          |
```

Comment:
Question: is it that Terraform Cloud doesn't _assign_ the policy set? Or that TFC doesn't _run_ the policy set for excluded workspaces? 

Small suggestion for active voice, but again not sure if this is correct without the above question:
```suggestion
| `data.relationships.exclusions`               | array\[object] | (previous value) | An array of references to workspaces that Terraform Cloud excludes from a policy set. Sending an empty array clears all exclusions assignments. You can only specify this attribute when `data.attributes.global` is `false`.                                                                                      |                                                                                                                                          |
```

Reply:
Its this one: Or that TFC doesn't run the policy set for excluded workspaces

Reply:
Thank you for clarifying! With that background,  I think this feels a little more clear (feel free to push back if you don't think so though!):
```suggestion
| `data.relationships.exclusions`               | array\[object] | (previous value) | An array of references to excluded workspaces that Terraform Cloud will not enforce this policy set upon. Sending an empty array clears all exclusions assignments. You can only specify this attribute when `data.attributes.global` is `false`.                                                                                      |                                                                                                                                          |
```

Reply:
I have changed it to: An array of references to excluded workspaces that the policy set should be assigned to.

Is this better?